doc/content/getting-started/migrating-from-v2/configure-mac-settings.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Configure End Devices
+weight: 60
+---
+
+{{< cli-only >}}
+
+This section is optional.
+
+MAC Parameters on {{% tts %}} are configurable per end device. After you have imported your end devices, you can use the CLI to configure their MAC settings.
```

Comment:
Isn't this generic for all devices, not just migrated devices?

Reply:
For ABP this has to be done during import, otherwise MAC settings are not effective

Reply:
@rvolosatovs Then we have to set this when exporting the devices from the v2? Is this documented somewhere? It would have been helpful to know this in advance.